### ISSUE TITLE
3.11: @import (less) broken

### ISSUE THREAD

**Issue by Diablohu** [less/less.js#3474]
im my less code:
```less
@import (less) "~normalize.css/normalize.css";
```

before 3.11: import `node_modules/normalize.css/normalize.css` as less code into my less file
after 3.11: Less try to find file `node_modules/normalize.css/normalize.css.less` and because this file does noe exist, building failed

Here's the error info from Webpack
```
@import (less) "~normalize.css/normalize.css";
^
Can't resolve 'normalize.css/normalize.css.less' in 'L:\Projects\koot\master\test\projects\simple\src'
      in L:\Projects\koot\master\test\projects\simple\src\critical.less (line 1, column 0)
    at L:\Projects\koot\master\node_modules\webpack\lib\NormalModule.js:316:20
    at L:\Projects\koot\master\node_modules\loader-runner\lib\LoaderRunner.js:367:11
    at L:\Projects\koot\master\node_modules\loader-runner\lib\LoaderRunner.js:233:18
    at context.callback (L:\Projects\koot\master\node_modules\loader-runner\lib\LoaderRunner.js:111:13)
 @ ./src/critical.js 9:2-28
```

**Comment by turbidwater** [comment]
We're seeing this issue too. Changelogs haven't been updated yet, so no word on if there was a syntax or functionality change.

**Comment by martinpitt** [comment]
This broke 3 of our projects as well. We [dropped the markers](https://github.com/cockpit-project/cockpit/commit/65dfb4fc177), I suppose if a file really uses less syntax, it should be called *.less. So it's mostly a little inconvenience.

**Comment by matthew-dean** [comment]
Please see if this is fixed on 3.11.1

**Comment by Diablohu** [comment]
3.11.1 solved this problem for me
thanks!
